driver-core/src/test/java/com/datastax/driver/core/SegmentBuilderTest.java

```diff
@@ -35,7 +35,7 @@
 public class SegmentBuilderTest {
 
   private static final Message.ProtocolEncoder REQUEST_ENCODER =
-      new Message.ProtocolEncoder(ProtocolVersion.V5);
+      new Message.ProtocolEncoder(ProtocolVersion.V6);
```

Comment:
We could have left V5 here imo as it is more important to test the encoding of segments in V5 than in V6. Same in `SegmentToFrameDecoderTest`.

Reply:
You're right, I'll revert it.

Reply:
Thanks! Would you mind reverting `SegmentToFrameDecoderTest` as well?